salt/metalk8s/addons/ui/config/metalk8s-shell-ui-config.yaml.j2

```
@@ -0,0 +1,28 @@
+#!jinja|yaml
+
+# Defaults for shell UI configuration
+apiVersion: addons.metalk8s.scality.com
+kind: Metalk8sShellUIConfig
```

We're already in the `addons.metalk8s.scality.com` API group, so I think this is enough:

```suggestion
kind: ShellUIConfig
```